scripts/_compile/index.js

```diff
@@ -112,21 +118,23 @@ async function compileScss () {
 
 async function compileScripts () {
     console.log('SRC: Compiling JavaScript');
-
-    // TODO: convert to npm script
-    await exec(`yarn rollup`, { cwd: CONFIG.root }, function(error, stderr){
```

Comment:
Changed command from `yarn rollup` to `yarn gulp bundle --prod`.

TODO: need to double check that compilation is going to be quick enough to _not_ be annoying while running `yarn start`.

Reply:
Also, modified arguments to the callback (second arg is stdout, not stderr).

Reply:
TODO: revert change to command. We need to use tweaked config with old set of configs to prevent adversely affecting the dev experience.